practitioner-service/src/main/resources/bootstrap.yml

```diff
@@ -1,6 +1,15 @@
 spring:
   application:
     name: practitioner-service
+  profiles: default
```

Comment:
we dont have a default profile/can this be a placeholdeR?

Reply:
Any bean that does not specify a profile belongs to the default profile.
We have a default profile out of the box. The thing that helped solving the docker-compose problem was explicitly stating the profile name.

Reply:
After talking in person realized I had a misconception of the profiles, fixed and was removed from file